### ISSUE TITLE
Ability to select multiple weight limit signs

### ISSUE THREAD

**Issue by vfosnar** [streetcomplete/StreetComplete#5803]
**Use case**
At least here in Czech Republic it's common to have 2 signs describing weight limit for different types of vehicles.

**Proposed Solution**
Add option to add more than one weight limit sign

**Comment by westnordost** [comment]
Can you post a few pictures?

**Comment by vfosnar** [comment]
It's usually these two on a single pole
![B13](https://github.com/user-attachments/assets/78641bc9-5938-4a87-972a-4f1fde027aff)
![B14](https://github.com/user-attachments/assets/010c1251-ecc0-4426-9237-a6209ef79303)

**Comment by westnordost** [comment]
I meant photos, but okay.

To gauge whether it is worth the time, effort and added UI complexity to implement this as opposed to let users leave a note (with photo) instead, we need to know how often this actually occurs, we need some numbers. How many notes are created for the weight limit quest? On how many bridges we have multiple different maxweight restrictions?

**Comment by vfosnar** [comment]
> I meant photos, but okay.

got that, just don't have a photo and couldn't find any on the internet

> How many notes are created for the weight limit quest?

not sure but overall weight limits are heavily underreported here in ČR

I can extract some statistics when I get home

**Comment by kmpoppe** [comment]
I might be stating the obvious, but that's not a different `maxweight` information, the second sign is `maxaxleload`.

These two signs have to be adhered to in conjunction, so in the case you've shown, your motorcar may only weight 5.5t tonnes, but if it only got 2 axles, there's no scenario in which the axleload isn't above 2t, so you may not cross that bridge (or whatever the signs are displayed at).

So, the question becomes whether there needs to be another quest asking whether the axleload is signed at all (and if yes, then what is it, much like the `maxheight` quest)?

**Comment by westnordost** [comment]
No, in the maxweight quest, one can select which there is. There are different signs to select from in the UI.

**Comment by vfosnar** [comment]
![20240823_193720_lmc_8 4](https://github.com/user-attachments/assets/af12b56f-2e4b-4603-8a77-3a5c90ba44ab)

**Comment by vfosnar** [comment]
also it seems like there is already an overlay in SCEE that can add additional road restrictions to existing ones but I'd still want to see this as a part of the quest

![Screenshot_20240823-193830_SCEE](https://github.com/user-attachments/assets/9b10793e-ca5a-4b6c-a43b-26c4ed5d292c)

**Comment by vfosnar** [comment]
Here in Czechia

**Comment by vfosnar** [comment]
Sorry my phone froze after loading an overpass query and ghost pressed the 'close with comment' button :D

for Czechia:
- 285 both maxaxleload and maxweight
- 312 only maxaxleload
- 8806 only maxweight

so maxaxleload is in ~91% times tagged in this combination and maxweight in ~3%

for the record I didn't see this combination tagged wrongly as 'maxweight only' yet, but I also don't actively check already mapped access restrictions

**Comment by westnordost** [comment]
Hm, well. That doesn't sound worth it to implement (, so I won't).

But as long as the UI  would look like the following, I would merge a PR:

Add "other answer" option for that there are multiple signs. This adds a plus button of sorts below the previously selected sign with which the user can select yet another sign (and another sign, another sign, another sign,....)

**Comment by RubenKelevra** [comment]
Not super common, but especially on old bridges you may see these signs also in Germany:

![achslast-verkehrsschild-1](https://github.com/user-attachments/assets/91992733-c049-4ab2-883d-6b85174e0776)


I have a hunch that people using SC would enter them as normal weight limits :/

**Comment by westnordost** [comment]
I don't think so. This is one of the signs that can be selected.
